src/components/Dashboard/TodoList.vue

```diff
@@ -294,7 +316,13 @@
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
-          <div v-if="isTypeCorrection(task)">
+          <v-card v-if="isBcolError(task)" class="todo-list-detail bcol-todo-list-detail">
+            <v-card-text>
+            <p class="font-weight-bold black--text">Payment Incomplete - {{task.bcolErrObj.title}}</p>
+            <span v-html="task.bcolErrObj.detail"/>
```

Comment:
As there are no checks anywhere else that bcolErrorObj is not null, please add a check here before de-referencing detail.

Reply:
done

Reply:
Fixed, although I think we should have a fallback if this is a BCOL error but we don't have the error object or title or details -- see Ody's comment above.